internal/distro/fedora31/distro.go

```diff
@@ -207,6 +207,7 @@ func New() *Fedora31 {
 			"cloud-init",
 			"checkpolicy",
 			"net-tools",
+			"network-scripts",
```

Comment:
In the linked RHBZ they talk about adding NetworkManager, but in the github issue, you explain that NM breaks cloud-init. I guess in the end they ended up with network-scripts instead. I think a link to the fedora31 qcow2 .ks as the authorotative source would be good for this PR. Also: test cases must be regenerated.

Reply:
Yup, I've re-read the bugs and removed that adjustment now.